### ISSUE TITLE
[FEATURE] A word: reference that links not to the whole article, but to the specified part of it

### ISSUE THREAD

**Issue by soshial** [goldendict/goldendict#348]
There are some word articles that are too long, so in XDXF format еhere is a feature of referencing a specific part of word article.

So, imagine we have a referencing article and a referenced one.
1. Referencing word article:

``` xml
<ar f="l">
    <k>inngangur</k>
    <def>
        <def>blah-blah1, <kref idref="q1w2e3r4t5y6">innilegur</kref></def>
    </def>
</ar>
```
1. Referenced word article  

``` xml
<ar f="l">
    <k>innilegur</k>
    <def>
        <def><dtrn>hluboký, vroucí</dtrn><i> (hjartanlegur)</i></def>
        <def><dtrn>milující, láskyplný</dtrn><i> (ástúðlegur)</i></def>
        <def id="q1w2e3r4t5y6"><dtrn>upřímný</dtrn><i> (einlægur)</i></def>
    </def>
</ar>
```

As you can see, the id is `q1w2e3r4t5y6`. What i intend to do is to place these CSS properties:

```
.xdxf_def:target
{
    border-color: red;
    border-width: 1px;
    border-style: double;
}`
```

So, logically, Goldendict is supposed to mark out the referenced part of `innilegur` article. But when I click clicked the link it doesn't trigger the `target` CSS pseudo-class. The output of Goldendict says that `clicked  QUrl( "bword:innilegur#test" )`.

May I ask you to help me to investigate what could be the reason for that. Thank you.

**Comment by nikita-moor** [comment]
Bring up the issue because I want to implement inter-dictionary links to particular parts of the articles.

**Comment by soshial** [comment]
Hey, @nikita-moor, did I get it correct, that you were going to implement this feature in Goldendict?
This referencing is already a part of the standard.

**Comment by nikita-moor** [comment]
No, I am not a software developer here, but a dictionary compiler. I am making a XDXF dictionary (of Latin language) and need the feature of referencing articles. Currently, I am forced to use HTML references, such as `<a href="car">jeep</a>`, which is not always good.
